hawkbit-security-core/src/main/java/org/eclipse/hawkbit/security/SystemSecurityContext.java

```diff
@@ -111,6 +117,44 @@ public SystemSecurityContext(final TenantAware tenantAware) {
         }
     }
 
+    /**
+     * Runs a given {@link Callable} within a system security context, which has
+     * the provided {@link GrantedAuthority}s to successfully run the
+     * {@link Callable}.
+     * 
+     * The security context will be switched to the a new
+     * {@link SecurityContext} and back after the callable is called.
+     * 
+     * @param tenant
+     *            to act as system code
```

Comment:
"system code" sounds wrong

Reply:
Inspired from "runAsSystemAsTenant".  Will remove it.